src/TestReporter/Entity/CiInfo.php

```diff
@@ -4,149 +4,169 @@
 class CiInfo
 {
     /**
+     * @var array
+     */
+    private $info;
+
+    public function __construct(array $server)
```

Comment:
Requiring `$server` here allows us to properly test it, and frees this class from needing to access global state.